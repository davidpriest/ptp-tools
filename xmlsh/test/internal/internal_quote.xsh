# Test of quote function

echo quote( 'hi"there' )
echo quote( <[ <foo>bar</foo> ]> )
echo quote( a b c <[ 123 ]> <[ <foo a="b"/> ]> )
echo quote( a\\b )

 