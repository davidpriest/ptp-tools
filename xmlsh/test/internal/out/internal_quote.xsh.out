"hi\"there"
"<foo>bar</foo>"
"abc123<foo a=\"b\"/>"
"a\\b"
